src/sbt-test/sbt-header/simple/build.sbt

```diff
@@ -0,0 +1,21 @@
+import de.heikoseeberger.sbtheader.license.Apache2_0
+
+headers := Map(
+  "scala" -> Apache2_0("2015", "Heiko Seeberger")
+)
+
+TaskKey[Unit]("check") <<= scalaSource in Compile map { (scalaSource) =>
```

Comment:
Is there a better way to compare file contents? Maybe some sbt API I'm not aware of?